playbooks/pki.yml

```diff
@@ -157,7 +158,7 @@
             state: directory
             mode: "{{ item.mode | default(omit) }}"
           loop:
-            - name: /etc/pki
+            - name: /etc/pki/{{ inventory_hostname | dirname }}
```

Comment:
Here I would not create subdirs and use `inventory_hostname | basename` when generating the key instead. This path is not reused anywhere else IIRC and thats rather unusual to have subdirs on the target host pki directory (even more becaue those subdir probably wil not make any sense at the host level)

Reply:
I reverted that part and use the basename on key creation.